make.ps1

```diff
@@ -114,8 +114,7 @@ function Check-Command
 {
 	Write-Host "Compiling in Debug configuration..." -ForegroundColor Cyan
 
-	# Enabling EnforceCodeStyleInBuild and GenerateDocumentationFile as a workaround for some code style rules (in particular IDE0005) being bugged and not reporting warnings/errors otherwise.
-	dotnet build -c Debug --nologo -warnaserror -p:TargetPlatform=win-x64 -p:EnforceCodeStyleInBuild=true -p:GenerateDocumentationFile=true
+	dotnet build -c Debug --nologo -warnaserror -p:TargetPlatform=win-x64
```

Comment:
~~Running `make.ps1 check` or this directly doesn't produce a warning or an error when there is an unused `using` statement now!~~
Not sure if related to the PR, but all of these, including `make check` only report warnings if doing a new build. We will need to make `make check` do a `clean` first!

Reply:
Nice spot, added.

Reply:
Some rules seem to work regardless of clean, but some don't. Could've easily been broken for a while, but CI is always a fresh build so was likely scooping up the leftovers.